web/src/views/DandisetLandingView/DandisetMain.vue

```diff
@@ -111,7 +145,59 @@
       </v-row>
 
       <v-divider />
+      <v-row
+        class="mx-2"
+        align="center"
+      >
+        <v-col>
+          <span
+            v-for="author in contributors"
+            :key="author.name + author.identifier"
+          >
+            <a
+              :href="author.identifier"
+              target="_blank"
+            >
+              <img
+                alt="ORCID logo"
+                src="https://info.orcid.org/wp-content/uploads/2019/11/orcid_16x16.png"
+                width="16"
+                height="16"
+              ></a>
+            <v-tooltip
+              v-if="author.affiliation"
+              top
+              color="black"
+            >
+              <template #activator="{ on }">
+                <span v-on="on">
+                  {{ author.name }}
+                </span>
+              </template>
+              <span>{{ author.affiliation }}</span>
+            </v-tooltip>
+            <span v-else> {{ author.name }}</span>
+            ;
+          </span>
+        </v-col>
+      </v-row>
+      <v-row :class="titleClasses">
+        <v-card-title class="font-weight-regular">
+          Keyword(s)
+        </v-card-title>
+      </v-row>
```

Comment:
remove mention of keywords
```suggestion
      <v-row :class="titleClasses">
        <v-card-title class="font-weight-regular">
          Keyword(s)
        </v-card-title>
      </v-row>
```